packages/scene-composer/src/components/panels/SettingsPanel.tsx

```diff
@@ -100,26 +101,41 @@ export const SettingsPanel: React.FC<SettingsPanelProps> = ({ valueDataBindingPr
         })}
         defaultExpanded
       >
-        <ComponentVisibilityToggle
-          componentType={KnownComponentType.MotionIndicator}
-          label={intl.formatMessage(visibilityToggleLabels[KnownComponentType.MotionIndicator])}
-        />
-        <ComponentVisibilityToggle
-          componentType={KnownComponentType.Tag}
-          label={intl.formatMessage(visibilityToggleLabels[KnownComponentType.Tag])}
-        />
-        {overlayEnabled && (
+        <FormField
+          label={
+            <Box fontWeight='bold'>
+              {intl.formatMessage({ defaultMessage: 'Toggle visibility', description: 'FormField label' })}
+            </Box>
+          }
+        >
           <ComponentVisibilityToggle
-            componentType={Component.DataOverlaySubType.OverlayPanel}
-            label={intl.formatMessage(visibilityToggleLabels[Component.DataOverlaySubType.OverlayPanel])}
+            componentType={KnownComponentType.MotionIndicator}
+            label={intl.formatMessage(visibilityToggleLabels[KnownComponentType.MotionIndicator])}
           />
-        )}
-        {overlayEnabled && (
+          <Divider />
           <ComponentVisibilityToggle
-            componentType={Component.DataOverlaySubType.TextAnnotation}
-            label={intl.formatMessage(visibilityToggleLabels[Component.DataOverlaySubType.TextAnnotation])}
+            componentType={KnownComponentType.Tag}
+            label={intl.formatMessage(visibilityToggleLabels[KnownComponentType.Tag])}
           />
-        )}
+          {overlayEnabled && (
+            <>
+              <Divider />
+              <ComponentVisibilityToggle
+                componentType={Component.DataOverlaySubType.OverlayPanel}
+                label={intl.formatMessage(visibilityToggleLabels[Component.DataOverlaySubType.OverlayPanel])}
+              />
+            </>
+          )}
+          {overlayEnabled && (
+            <>
+              <Divider />
+              <ComponentVisibilityToggle
+                componentType={Component.DataOverlaySubType.TextAnnotation}
+                label={intl.formatMessage(visibilityToggleLabels[Component.DataOverlaySubType.TextAnnotation])}
+              />
```

Comment:
Nit: Could you ensure there are descriptions for all translations to make them easier to find?

Reply:
yeah, descriptions are in `visibilityToggleLabels`